core-meta.html

```diff
@@ -68,12 +68,54 @@
          * @type String
          * @default 'default'
          */
-        type: String
+        type: String,
+
+        /**
+         * The value of the meta-data to store. If no value is specified, the
+         * value defualts to a self reference (`this`).
+         *
+         * @attribute value
+         * @type *
+         * @default this
+         */
+        value: String,
+
+        /**
+         * The key used to store the `value` under the `type` namespace.
+         * Defaults to the `id` value.
+         *
+         * @attribute key
+         * @type String
+         * @default this.id
+         */
+        key: String
```

Comment:
When talking to @morethanreal about the change before I implemented it, she expressed interest in keeping `id`. I would like to get her go-ahead before removing it entirely.